packages/commerce-sdk-react/src/hooks/useQuery.ts

```diff
@@ -48,7 +48,17 @@ export const useQuery = <Client extends ApiClient, Options extends ApiOptions, D
     // missing parameters. This will result in a runtime error. I think that this is an acceptable
     // trade-off, as the behavior is opt-in by the end user, and it feels like adding type safety
     // for this case would add significantly more complexity.
-    const wrappedMethod = async () => await authenticatedMethod(apiOptions as Options)
+    const wrappedMethod = async () => {
```

Comment:
Can we add test cases to test the error please?

Reply:
yep, i'm working on it